### PR TITLE
ls: pt-BR improving descriptions

### PR DESCRIPTION
**Pull request by andrik**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

Hi guys, I'm submitting this PR to address the suggestions without feedback on #3396

### CODE REVIEW

File: pages.pt_BR/common/ls.md [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
 
 `ls -a`
 
-- Listar (formato longo) todos os arquivos (permissões, propriedade, tamanho e data de modificação):
+- Listar todos os arquivos, exibindo permissões, propriedade, tamanho e data de modificação:
 
 `ls -la`
 
-- Listar (formato longo) todos os arquivos, apresentando o tamanho usando medidas legíveis por humanos (KB, MB, GB).
+- Listar todos os arquivos, apresentando o tamanho em medidas legíveis por humanos (KB, MB, GB).
```
Comment (zdroid) [inline pages.pt_BR/common/ls.md]: Maybe `kB` instead of `KB`? (this should be changed in English aswell)

**Comment by zdroid** [comment]
:+1: Do it in pt_BR version too.

**Comment by andrik** [comment]
My mistake, I forgot to select pt-BR version :man_facepalming:

**Comment by zdroid** [comment]
I meant to keep that one... ehh. It should be changed both in original and in the pt_BR version.

**Comment by andrik** [comment]
I'll create another PR to standardize the kilobytes symbol on other versions

**Pull request merged**